### PR TITLE
Add signature component

### PR DESCRIPTION
**Pull request by ToddWebDev**
## Summary
Update the 'Review application' page in the Pension Benefits application(21P-527EZ) to include the Signature component

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Sign in via mocked authentication at `http://localhost:3001/sign-in/mocked-auth`
3. Start a new application
4. Fill out the 'Applicant information' page including first name, middle name(optional) and last name
5. Open the 'Save in progress' menu
6. Select 'Review and submit' as the return url
7. Go to the 'Review application' page (/review-and-submit)
8. Verify that the 'Statement of Truth' is now at the end of the form instead of the 'privacy' checkbox
9. Verify that both fields in the 'Statement of Truth' are required
10. Verify that the full name must match the entered applicant's name
11. Enter the veteran's full name and check the 'certification' checkbox
12. Verify that all chapters and pages are validated before submitting the form

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73016)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done
- [ ] Existing unit tests updated
- [ ] New unit tests added

## Screenshots
| Before | After | Invalid | Valid |
| -------- | -------- | -------- | -------- |
|localhost_3001_pension_application_527EZ_review-and-submit (3)|localhost_3001_pension_application_527EZ_review-and-submit|localhost_3001_pension_application_527EZ_review-and-submit (1)|
localhost_3001_pension_application_527EZ_review-and-submit (2)

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73016)

### Quality Assurance & Testing
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/toe/components/IntroductionLogin.jsx [hunk 1]
```diff
@@ -18,15 +18,12 @@ function IntroductionLogin({
   isLOA3,
   isPersonalInfoFetchComplete,
   isPersonalInfoFetchFailed,
-  isSponsorsFetchComplete,
```
Comment (TaiWilkin) [inline src/applications/toe/components/IntroductionLogin.jsx:21]: Are these changes related to the PR? I'm having trouble tracking down where we use this in Pensions

Comment (ToddWebDev) [reply src/applications/toe/components/IntroductionLogin.jsx]: No, that is an unrelated change based on a previous base. That change should no longer be included in the file changes.

**Review by TaiWilkin — DISMISSED** [review]
LGTM

**Review by TaiWilkin — APPROVED** [review]
Still working well after base change
